trunk/SUAVE/Methods/Weights/Buildups/eHelicopter/fuselage.py

```diff
@@ -0,0 +1,118 @@
+# fuselage.py
+#
+# Created: Jun 2017, J. Smart
+
+#-------------------------------------------------------------------------------
+# Imports
+#-------------------------------------------------------------------------------
+
+from SUAVE.Core import Units
+from SUAVE.Attributes import Solids                     
+import numpy as np
+
+
+#-------------------------------------------------------------------------------
+# Fuselage
+#-------------------------------------------------------------------------------
+
+def fuselage(fLength, fWidth, fHeight, maxSpan, MTOW):
+    """ weight = SUAVE.Methods.Weights.Correlations.eHelicopter.fuselage(
+            fLength,
+            fWidth,
+            fHeight,
+            maxSpan,
+            MTOW
+        )
+
+        Calculates the structural mass of a fuselage for an eVTOL vehicle,
+        assuming a structural keel taking bending an torsional loads.
+
+        Assumes an elliptical fuselage. Intended for use with the following
+        SUAVE vehicle types, but may be used elsewhere:
+
+            eHelicopter
+            eTiltwing
+            eTiltrotor
+            eStopped_Rotor
+
+        Originally written as part of an AA 290 project intended for trade study
+        of the above vehicle types.
+
+        Inputs:
+
+            fLength:    Fuselage Length     [m]
+            fWidth:     Fuselage Width      [m]
+            fHeight:    Fuselage Height     [m]
+            maxSpan:    Maximum Wingspan    [m]
+            MTOW:       Max TO weight       [kg]
+
+        Outputs:
+
+            weight:     Fuselage Mass   [kg]
+    """
+
+    G_max   = 3.8    # Maximum G's Experienced During Climb
```

Comment:
Make these accessible to the user. Use defaults or inputs to the function.

Reply:
Changed to function input with default value.